packages/client/lib/net/protocol/snapprotocol.ts

```diff
@@ -1,19 +1,30 @@
-import { bigIntToBuffer, bufferToBigInt } from '@ethereumjs/util'
+import {
+  accountBodyFromSlim,
+  accountBodyToSlim,
+  bigIntToBuffer,
+  bufferToBigInt,
+  setLengthLeft,
+} from '@ethereumjs/util'
 
 import { Protocol } from './protocol'
 
 import type { Chain } from '../../blockchain'
 import type { Message, ProtocolOptions } from './protocol'
-import type { AccountData as AccountDataBody } from '@ethereumjs/util'
+import type { AccountBodyBuffer } from '@ethereumjs/util'
 
 interface SnapProtocolOptions extends ProtocolOptions {
   /* Blockchain */
   chain: Chain
+  /**
+   * If to convert slim body received of an account to normal while decoding.
+   * Encoding is always converted to slim
+   */
+  convertSlimBody?: boolean
```

Comment:
Why would we want to keep the slim body?

Reply:
is slim body not useful at all? then we can always convert and remove this flag totally